text.cabal

```diff
@@ -94,6 +94,12 @@ library
       extra-libraries: c++
     elif os(openbsd)
       extra-libraries: c++ c++abi pthread
+    elif os(windows) && impl(ghc > 9.3)
+      extra-libraries: c++ c++abi
+    elif arch(wasm32)
+      cpp-options: -DSIMDUTF_NO_THREADS
+      cxx-options: -fno-exceptions
+      extra-libraries: c++ c++abi
```

Comment:
You can workaround `arch(wasm32)` by swapping branches:
```cabal
    elif arch(linux)
      extra-libraries: stdc++
    else 
      cpp-options: -DSIMDUTF_NO_THREADS
      cxx-options: -fno-exceptions
      extra-libraries: c++ c++abi
```
Actually this is no worse than before: it's not like `extra-libraries: stdc++` is correct for, say, `arch(netbsd)` or other exotic systems.